js/api.js

```diff
@@ -13,7 +12,7 @@ function getData(makePins, onFail) {
       }
     })
     .then((response) => response.json())
-    .then((data) => makePins(data.slice(0, COUNT_ADVERTS)))
+    .then((data) => onSucess(data))
```

Comment:
```suggestion
    .then(onSucess)
```

зачем создавать новую функцию, что просто вызовет другую? Передай эту другую как коллбек и все.

Reply:
поправил